axelor-account/src/main/java/com/axelor/apps/account/service/YearServiceAccountImpl.java

```diff
@@ -249,4 +253,26 @@ public BigDecimal computeReportedBalance2(
     }
     return reportedBalanceAmount;
   }
+
+  /**
+   * function which retrieve Year Object for the Calendar year current and the type of Year
+   *
+   * @return
+   */
+  public Year getYear(int intTypeSelect) {
```

Comment:
This is duplicated code, either remove this or remove the method in PartnerTurnoverServiceImpl.java line 216

Reply:
Done